travis/apis.py

```diff
@@ -54,6 +56,18 @@ def __init__(self):
         self.ssh = os.environ.get(
             "WEBLATE_SSH", "ssh://user@webpage.com")
         self.tempdir = os.path.join(tempfile.gettempdir(), 'weblate_api')
+        self._ssh_keyscan()
+
+    def _ssh_keyscan(self):
+        cmd = "ssh-keyscan -p %s %s"
+        match = re.search(r'@(?P<port_host>[^/]+)', self.ssh)
```

Comment:
Check runbot comments https://github.com/Vauxoo/runbot-addons/pull/141#discussion_r138675892 (ignore LRU CACHE here)

Reply:
@moylop260 
I applied the changes I could apply to this code